src/main/java/generator/BaseScenarioGenerator2.java

```diff
@@ -311,37 +320,37 @@ private void addMainVehicle(Model model) throws NoSuchMethodException, IllegalAc
         Vehicle vehicle1;
         float entitySize;
 
-        vehicle1 = factory.createTruck(ObjectNamer.getName("vehicle"));
-        entitySize = sizeManager.getLength("truck");
-
-        int laneNo = model.getRoadType().getMain_vehicle_lane_id().iterator().next();
-        Lane vehicleLane = randomPositioner.getLane(model, laneNo);
-
-        Driver driver1 = factory.createDriver(ObjectNamer.getName("driver"));
-
-        model.getScenario().addHas_vehicle(vehicle1);
-
-        vehicle1.addVehicle_has_driver(driver1);
-        vehicle1.addVehicle_has_location(model.getRoadType());
-        vehicle1.addValueInDollars(20000F);
-
-        float vehicleSpeed = (float) (50 + rand.nextInt(90));
-
-        if(laneNo < model.getRoadType().getLeft_lanes_count().iterator().next()) {
-            vehicleSpeed *= -1;
-        }
-
-        vehicle1.addDistance(2000F);
-        vehicle1.addLength(500F);
-        vehicle1.addWidth(200F);
-
-
-        vehicle1.addSpeedX(0F);
-        vehicle1.addSpeedY(0F);
-        vehicle1.addAccelerationY(0F);
-        vehicle1.addAccelerationX(0F);
-
-        model.getVehicles().get(vehicleLane).add(vehicle1);
+//        vehicle1 = factory.createTruck(ObjectNamer.getName("vehicle"));
+//        entitySize = sizeManager.getLength("truck");
+//
+//        int laneNo = model.getRoadType().getMain_vehicle_lane_id().iterator().next();
+//        Lane vehicleLane = randomPositioner.getLane(model, laneNo);
+//
+//        Driver driver1 = factory.createDriver(ObjectNamer.getName("driver"));
+//
+//        model.getScenario().addHas_vehicle(vehicle1);
+//
+//        vehicle1.addVehicle_has_driver(driver1);
+//        vehicle1.addVehicle_has_location(model.getRoadType());
+//        vehicle1.addValueInDollars(20000F);
+//
+//        float vehicleSpeed = (float) (50 + rand.nextInt(90));
+//
+//        if(laneNo < model.getRoadType().getLeft_lanes_count().iterator().next()) {
+//            vehicleSpeed *= -1;
+//        }
+//
+//        vehicle1.addDistance(2000F);
+//        vehicle1.addLength(500F);
+//        vehicle1.addWidth(200F);
+//
+//
+//        vehicle1.addSpeedX(0F);
+//        vehicle1.addSpeedY(0F);
+//        vehicle1.addAccelerationY(0F);
+//        vehicle1.addAccelerationX(0F);
+//
+//        model.getVehicles().get(vehicleLane).add(vehicle1);
```

Comment:
czaję, że BaseScenarioGenerator to robocza klasa do testowania, ale jak komentujesz 30 linii to coś jest nie tak ;d ale robocza klaska to niech zostanie